components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -1149,10 +1158,23 @@ public DataSection medicalHistoryBody(Patient patient)
 
         Integer x = 0;
         if (present.contains("global_age_of_onset")) {
-            PatientData<SolrOntologyTerm> qualifiers = patient.getData("global-qualifiers");
-            SolrOntologyTerm ageOfOnset = qualifiers != null ? qualifiers.get("global_age_of_onset") : null;
-            DataCell cell = new DataCell(ageOfOnset != null ? ageOfOnset.getName() : "", x, 0);
-            bodySection.addCell(cell);
+            PatientData<List<SolrOntologyTerm>> qualifiers = patient.getData("global-qualifiers");
+            List<SolrOntologyTerm> ageOfOnsetList = qualifiers != null ? qualifiers.get("global_age_of_onset") : null;
+            // DataCell cell = new DataCell(ageOfOnset != null ? ageOfOnset.getName() : "", x, 0);
+            // bodySection.addCell(cell);
+            // x++;
+            int y = 0;
+            if (ageOfOnsetList != null && !ageOfOnsetList.isEmpty()) {
+                for (SolrOntologyTerm term : ageOfOnsetList) {
+                    String mode = term != null ? term.getName() : "";
```

Comment:
`mode` is not the right variable name, this is about ages, not modes of inheritance.